src/main/resources/CodyBundle.properties

```diff
@@ -68,3 +68,16 @@ context-panel.error-message=Cody 'Keyword Search' indexer failed: {0}
 context-panel.help-button-name=Help
 context-panel.tree-root=repositories-root
 context-panel.tree-local-project=<b>1 local project</b>
+
+EndOfTrialNotification.link-action-name= Setup Payment Info
+EndOfTrialNotification.do-not-show-again=Don't show again
+
+TrialEndingSoonNotification.ignore=cody.ignore.notification.trial-ending-soon
+TrialEndingSoonNotification.ending-soon.title=Your Cody Pro trial is ending soon
+TrialEndingSoonNotification.ending-soon.content=Setup your payment information to continue using Cody Pro, you won't be charged until February 15.
+TrialEndingSoonNotification.ending-soon.link=https://accounts.sourcegraph.com/cody/subscription?on-trial=true
+
+TrialEndedNotification.ignore=cody.ignore.notification.trial-ended
+TrialEndedNotification.ended.title=Your Cody Pro trial has ended
+TrialEndedNotification.ended.content=<html>Thank you for using Cody Pro, you have been downgraded to Cody Free.<br>Setup your payment information to continue using Cody Pro for $9 / month, and cancel anytime.</html>
```

Comment:
@mkondratek  An alternative that is a bit more generic so that we can change price or currency without changing this notification

"Thank you for trying Cody Pro! Your trial period has ended. To keep enjoying all the features of Cody Pro, subscribe to our monthly plan. You can cancel anytime."

Reply:
good point! thanks 💟